plugins/optimization-detective/storage/rest-api.php

```diff
@@ -182,6 +188,7 @@ function od_handle_rest_request( WP_REST_Request $request ) {
 					// Now supply the readonly args which were omitted from the REST API params due to being `readonly`.
 					'timestamp' => microtime( true ),
 					'uuid'      => wp_generate_uuid4(),
+					'eTag'      => $request->get_param( 'eTag' ),
```

Comment:
This is good. It makes sense that the ETag wouldn't be provided with the URL Metric object in the JSON body since it's not something that the client collects from the document. In this way it is similar to the `uuid` and `url`, although in theory the `url` and `etag` could be provided with in the `urlMetric` object even though they will just be ignored.